### PR TITLE
More markdown conversion fixes

### PR DESCRIPTION
**Pull request by martijnbastiaan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,8 @@
-Clash - A functional hardware description language
-==================================================
+# Clash - A functional hardware description language
 
-[![Hackage Release Version](https://img.shields.io/hackage/v/clash-ghc.svg)](https://hackage.haskell.org/package/clash-ghc)
-[![Hackage Dependency Freshness](https://img.shields.io/hackage-deps/v/clash-ghc.svg?style=flat)](https://packdeps.haskellers.com/feed?needle=exact%3Aclash-ghc)
-[![Documentation Status](https://readthedocs.org/projects/clash-lang/badge/?version=latest)](https://clash-lang.readthedocs.io/en/latest/?badge=latest)
```
Comment (alex-mckenna) [inline README.md]: Do we not want this badge?

Comment (alex-mckenna) [reply README.md]: Super useful highlighting from GitHub. I mean the readthedocs badge

Comment (martijnbastiaan) [reply README.md]: We do! Sorry

Comment (alex-mckenna) [reply README.md]: Okay, if you add that back I'll approve it

Comment (martijnbastiaan) [reply README.md]: Done.

**Pull request merged**